tests/macros/fixtures/interface-overview/EventTarget.json

```diff
@@ -0,0 +1,467 @@
+[
```

Comment:
An alternative to these long fixture files would be to cut them way down to the minimum and just code them directly into the test file. But if you think that other tests might be able to use them as well, that is a good reason to use fixture files.

Reply:
The only thing I stripped from these files are non en‑US locales (e.g.:&nbsp;the&nbsp;content&nbsp;of&nbsp;`translations`).